open-reac/README.md

```diff
@@ -0,0 +1,3 @@
+# OpenReac
+A reactive optimal powerflow for better loadflow convergence.
```

Comment:
OpenReac is a reactive optimal power flow that gives a set of parameters for voltage and reactive controls by network equipments such as generators, shunt compensators and transformers. OpenReac is used for network planning.